library/goptions.ml

```diff
@@ -543,8 +546,15 @@ let print_tables () =
   let open Pp in
   let print_option key value depr =
     let msg = str "  " ++ str (nickname key) ++ str ": " ++ msg_option_value value in
-    if depr then msg ++ str " [DEPRECATED]" ++ fnl ()
-    else msg ++ fnl ()
+    let depr = pr_opt (fun depr ->
+        hov 2
+          (str "[DEPRECATED" ++
+           pr_opt (fun since -> str "since " ++ str since) depr.Deprecation.since ++
+           pr_opt str depr.Deprecation.note ++
```

Comment:
```suggestion
           pr_opt (fun since -> str " since " ++ str since) depr.Deprecation.since ++
           pr_opt (fun note -> str " " ++ stt note) depr.Deprecation.note ++
```